components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1292,11 +1303,24 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     }
 
     protected async internalGetWorkspace(id: string, db: WorkspaceDB): Promise<Workspace> {
-        const ws = await db.findById(id);
-        if (!ws) {
+        const workspace = await db.findById(id);
+        if (!workspace) {
             throw new ResponseError(ErrorCodes.NOT_FOUND, "Workspace not found.");
         }
-        return ws;
+        if (!workspace.organizationId && this.user?.additionalData?.isMigratedToTeamOnlyAttribution) {
+            try {
+                log.info({ userId: this.user.id }, "Updating workspace without orgId.");
+                const userOrg = await this.userToTeamMigrationService.getUserOrganization(this.user);
+                const latestInstance = await this.workspaceDb.trace({}).findCurrentInstance(workspace.id);
+                this.userToTeamMigrationService.updateWorkspacesOrganizationId(
```

Comment:
Missing `await`? Or is this supposed to be not awaited on? 
https://github.com/gitpod-io/gitpod/pull/17489 to fix the build